modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/RexUtils.java

```diff
@@ -1191,6 +1202,34 @@ public Void visitLocalRef(RexLocalRef locRef) {
         return val;
     }
 
+    /** Return literal holding java object according to it`s type. */
+    public static @Nullable Object getValueFromLiteral(NativeType physicalType, RexLiteral lit) {
```

Comment:
I'm wondering if we really need one more way to extract values from literal expressions. We already have `literalValue`+`TypeUtils#fromInternal`